guides/v2.2/config-guide/multi-master/multi-master_manual.md

```diff
@@ -258,7 +258,7 @@ mysqldump -u <your database root username> -p <your main magento DB name> sequen
 
 This script restores sales data in your quote database.
 
-#### NDB requirement
+#### NDB requirement {#sql-sales-restore-ndb-req}
```

Comment:
Why is this necessary?

Reply:
This topic has sections with the same name. The sections with the same name have the same ID by default, that causes a duplication ID error.